### PR TITLE
adds conversion to TFTA and TFT, closes #107

### LINKED ISSUE #107: modify the conversion service to tfta
tfta should be issued but according to the spec:
>time locked TFT on the rivine chain become time locked TFTA if the locktime expires before 2021. I the locktime expires after 2020, they become time locked TFT on the Stellar blockchain

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

**Comment by DylanVerstraete** [comment]
@robvanmieghem do unlocked tokens become TFTA as well?

**Comment by robvanmieghem** [comment]
@DylanVerstraete  yes, they become tfta

File: ThreeBotPackages/conversion-service/actors/conversion_service.py [hunk 1]
```diff
@@ -136,11 +141,21 @@ def format_output(lock_time, unlock_tx_xdr):
...
+                    break
+                lock_time_date = datetime.fromtimestamp(lock_time)
+                # if lock time year is before 2021 be convert to TFTA
+                if lock_time_date.year < 2021:
+                    asset = _TFTA_FULL_ASSETCODES[str(converter_wallet.network)]
+                # else we convert to TFT
+                else:
+                    asset = _TFT_FULL_ASSETCODES[str(converter_wallet.network)]
+
+                if time.time() < lock_time:
```
Comment (robvanmieghem) [inline ThreeBotPackages/conversion-service/actors/conversion_service.py:157]: else? is this possible?

Comment (DylanVerstraete) [reply ThreeBotPackages/conversion-service/actors/conversion_service.py:157]: Well unlocked coin outputs have a `lock_time` of 0. So yes this is possible

**Comment by DylanVerstraete** [comment]
> default asset is tfta and can you check my codecomment on line 157?

Done

**Pull request merged**